### PR TITLE
Enable audio 🔊

### PR DESCRIPTION
**Pull request by bengreenier**
Adds support for audio capture at the rtc level.

+ Refactored `createStream` argument types
+ Default to capturing audio
+ Non-configurable at this time (like video)

### CODE REVIEW

**Comment by bengreenier** [comment]
Marking invalid - awaiting manual testing (by me)

**Comment by bengreenier** [comment]
ah this is fun. PRs get blocked now by the quality gate, as it wants better code coverage

**Comment by bengreenier** [comment]
allowing bypass of SonarCloud for now, while we track and work on #19

File: src/index.ts [hunk 1]
```diff
@@ -104,104 +104,98 @@ export const forceH264Sdp = (sdp: any) => {
...
-            chromeMediaSourceId: source.id,
-          },
-        },
-      } as any);
+export const createStream = (source: DesktopCapturerSource, opts?: MediaStreamConstraints) => {
+  return navigator.mediaDevices.getUserMedia({
+    ...opts,
+    audio: {
+      mandatory: {
+        chromeMediaSource: "system",
```
Comment (phongcao) [inline src/index.ts]: Why would we use "system" instead of "desktop" here?

Comment (bengreenier) [reply src/index.ts]: I don't remember why, and can't find the context at this time 😞 

We can switch to "desktop" and test/maybe change, though we know this to be working currently...

**Comment by bengreenier** [comment]
I'm seeing a bug with this when testing the `desktop` flag, adding `invalid` while investigating.

**Comment by bengreenier** [comment]
@phongcao - do you still have the ability to debug with the electron source?

I'm trying to understand if we're hitting [this line](https://chromium.googlesource.com/external/webrtc/stable/webrtc/+/master/modules/desktop_capture/mouse_cursor_monitor_x11.cc#148) which will mean we can never hide the cursor, yet.

Related to getUserMedia vs getDisplayMedia so discussing here.

**Comment by bengreenier** [comment]
We'll close this and reopen a similar PR, given all the changes that would be needed to make this viable.

**Pull request closed without merging**